[v100a:01607] WARNING: local probe returned unhandled shell:unknown assuming bash
rm: cannot remove '/lib': Is a directory
python: can't open file 'run.py': [Errno 2] No such file or directory
python: can't open file 'run.py': [Errno 2] No such file or directory
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
python: can't open file 'run.py': [Errno 2] No such file or directory
python: can't open file 'run.py': [Errno 2] No such file or directory
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[13906,1],2]
  Exit code:    2
--------------------------------------------------------------------------
